lib/analysis_server.coffee

```diff
@@ -108,6 +81,7 @@ class AnalysisServer
       callback(obj.event, obj)
 
   setAnalysisRoots: (paths) =>
+    @currentAnalysisRoots = new Set(paths)
```

Comment:
Perhaps log whenever we set the analysis roots? I'm not positive, but I suspect that this is an expensive operation.